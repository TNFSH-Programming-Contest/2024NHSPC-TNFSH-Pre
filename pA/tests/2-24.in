375168776489087020 424020215045636720 244913957463545893
55502616181962521 182501256303386010 323061159028419682
