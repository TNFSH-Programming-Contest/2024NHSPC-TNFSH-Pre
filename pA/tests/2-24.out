1630454397833386170
